commits.go

```diff
@@ -247,3 +250,103 @@ func (s *CommitsService) PostCommitComment(
 
 	return c, resp, err
 }
+
+// GetCommitStatusesOptions represents the available GetCommitStatuses() options.
+//
+// GitLab API docs: http://doc.gitlab.com/ce/api/commits.html#get-the-status-of-a-commit
+type GetCommitStatusesOptions struct {
+	Ref   string `url:"ref,omitempty"`
+	Stage string `url:"stage,omitempty"`
+	Name  string `url:"name,omitempty"`
+	All   bool   `url:"all,omitempty"`
+}
+
+// CommitStatus represents a GitLab commit status.
+//
+// GitLab API docs: http://doc.gitlab.com/ce/api/commits.html#get-the-status-of-a-commit
+type CommitStatus struct {
+	ID          *int       `json:"id"`
+	SHA         *string    `json:"sha"`
+	Ref         *string    `json:"ref"`
+	Status      *string    `json:"status"`
+	Name        *string    `json:"name"`
+	TargetUrl   *string    `json:"target_url"`
+	Description *string    `json:"description"`
+	CreatedAt   *time.Time `json:"created_at"`
+	StartedAt   *time.Time `json:"started_at"`
+	FinishedAt  *time.Time `json:"finished_at"`
+	Author      *Author    `json:"author"`
+}
+
+// GetCommitStatuses gets the statuses of a commit in a project.
+//
+// GitLab API docs: http://doc.gitlab.com/ce/api/commits.html#get-the-status-of-a-commit
+func (s *CommitsService) GetCommitStatuses(
+	pid interface{}, sha string, opt *GetCommitStatusesOptions) ([]*CommitStatus, *Response, error) {
```

Comment:
_nitting_ in these cases I prefer to have a param per line. This this would be written as:

```go
func (s *CommitsService) GetCommitStatuses(
  pid interface{},
  sha string,
  opt *GetCommitStatusesOptions) ([]*CommitStatus, *Response, error) {
```

This is mainly nitting but would like to keep it consistent.